pkg/storage/db.go

```diff
@@ -15,7 +15,7 @@ import (
 )
 
 const (
-	defaultPartitionInterval   = 3 // months
+	defaultPartitionInterval   = 90 // days
```

Comment:
question: why not switch to the `time.Duration` type now that it's not fixed?

Reply:
I felt this was more readable than time.Hour * 2160. Also, we currently don't want any partial day partitions, so time.AddDate() seems to be a better fit than time.Add().